_people/1350_carla-bynum.md

```diff
@@ -0,0 +1,37 @@
+---
+# Page template info (DO NOT EDIT)
+layout: default
+banner_file: banner--people-projects-lg.svg
+banner_file_mobile: banner--people-projects-sm.svg
+people_page: true
+
+# Page link (Edit this)
+permalink: people/carla-bynum
+
+# People Carousel and Page (Edit this)
+name: Carla Bynum
+pronouns: She/Her
+alumni: false
+cop: Front Office
+city_of_origin: Washington, DC
+previously_at: U.S. General Services Administration
+image_name: carla-bynum.jpg
+quote: "I love that USDS is intentional about selecting projects and is dedicated to helping underrepresented communities, such as veterans and immigrants."
+
+---
+
+**What’s your background?**
+
+Prior to joining the USDS at DHS team, I provided administrative support for various offices within the General Services Administration. Nine years ago, I started out supporting the GSA American Recovery Act office where I worked in the Limited Scopes office. I spent a bulk of my time with GSA supporting their Field Office and Opportunities Center at St. Elizabeths campus where we provided workforce development opportunities to residents in Ward 8. I then moved to the GSA maintenance office at the Nebraska Avenue Complex.
+
+**What inspired you to join USDS?**
+
+Community-service has always been a passion of mine and I was heavily involved in community service and outreach efforts at my previous job. I was attracted to USDS because of the focus on people-centered development. I love that USDS is intentional about selecting projects and is dedicated to helping underrepresented communities, such as veterans and immigrants.
```

Comment:
```suggestion
Community service has always been a passion of mine, and I was heavily involved in community service and outreach efforts at my previous job. I was attracted to USDS because of the focus on people-centered development. I love that USDS is intentional about selecting projects and is dedicated to helping underrepresented communities, such as Veterans and immigrants.
```